### DISCUSSION TITLE
thoughts

### DISCUSSION THREAD

**Discussion by n13l420** [community/community#130294] [New to GitHub]
### Select Topic Area

Show & Tell

### Body

I truly enjoyed the easy, understandable fun way to learn github. It gave me a lot of motivation to continue my journey through out this platform and i am glad this happened.

**Comment by ghostinhershell** [comment]
Welcome to GitHub and the GitHub Community @n13l420 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

Best of luck!
